### ISSUE TITLE
Bug - File Upload- Inaccessible component

### ISSUE THREAD

**Issue by agagancarczyk** [patternfly/patternfly-react#11343]
**Describe the problem**
This is an issue with `@patternfly/react-core": "^5.4.10"`

Please give a clear and concise description of the problem. Which components are affected?
The component is missing aria-label, manually trying to add the aria-label is also not possible. The issue is critical from the accessibility point of view.

**How do you reproduce the problem?**
It is possible to reproduce it in the Keycloak Admin UI > Clients > Import Client Form by using the Deque tool.

**Expected behavior**
It is possible to add aria-label to make the FileUpload accessibile.

**Is this issue blocking you?**
Yes, we were forced to remove the a11y test (https://github.com/keycloak/keycloak/pull/35517), but we will need to add it in as soon as this is fixed.

**Screenshots**
![Image](https://github.com/user-attachments/assets/d6c62f05-82b8-4e08-b56d-9deaa0cf5df3)

**What is your environment?**
 - Chrome (latest)
